packages/cli/src/commands/create.ts

```diff
@@ -18,7 +18,7 @@ export default class Init extends Command {
   static examples = ['$ yarn faststore init discovery']
```

Comment:
```suggestion
  static examples = ['$ yarn faststore create discovery']
```

Reply:
I had missed the readme as well. Fixed it :pray:

Reply:
The readme is updated automatically on the publish step (`oclif readme` is run there)